### PR TITLE
Add release checklist

### PR DESCRIPTION
**Pull request by lognaturel**
Document some things that I do when preparing a release that might not be obvious.

### CODE REVIEW

**Comment by lognaturel** [comment]
There are starting to be lots of open PRs! @grzesiek2010, do you think you could give this and https://github.com/getodk/collect/pull/5067 a quick sanity check? I don't think either has anything controversial. 🙏

File: README.md [hunk 1]
```diff
@@ -233,22 +233,32 @@ This is encountered when Robolectric has problems downloading the jars it needs
...
+  RELEASE_STORE_FILE=/path/to/collect.keystore
+  RELEASE_STORE_PASSWORD=secure-store-password
+  RELEASE_KEY_ALIAS=key-alias
+  RELEASE_KEY_PASSWORD=secure-alias-password
+  ```
+
+- a `google-services.json` file in the `collect_app/src/odkCollectRelease` folder. The contents of the file are similar to the contents of `collect_app/src/google-services.json`.
+
+### Release checklist:
+- tag the build by [adding a release](releases).
```
Comment (grzesiek2010) [inline README.md]: This link doesn't seem to work

**Review by grzesiek2010 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lognaturel) [reply README.md]: It's a relative link that would work once the PR is merged. But I guess in this case there's only one set of releases so I've changed it to an absolute link.

Comment (grzesiek2010) [reply README.md]: > It's a relative link that would work once the PR is merged.

But the one we already have in https://github.com/getodk/collect doesn't work as well. The absolute one seems to work well.

**Review by grzesiek2010 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lognaturel) [reply README.md]: Oh, well then! Thanks for the fix.

**Pull request merged**